sonarqubeandjacoco/src/main/java/com/baeldung/codecoverage/sonarqubeandjacoco/App.java

```diff
@@ -0,0 +1,13 @@
+package com.baeldung.codecoverage.sonarqubeandjacoco;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
+{
+    public static void main( String[] args )
+    {
```

Comment:
Opening curly braces go at the end of the previous line, not on a line by themselves.